### PR TITLE
ENDOC-640 Entando Entando Identity Management System Documentation

### PR DESCRIPTION
**Pull request by Lyd1aCla1r3**

### CODE REVIEW

File: vuepress/docs/next/docs/consume/identity-management.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Keycloak authorizes microservices using clients and roles. Authorizations are st
 
 Below are the steps to grant a user one or more roles for a specific client. This controls permissions when configuring the microservice. Note: when a microservice is installed in Entando, a corresponding client (and set of roles) is created per its plugin definition.
 
-1. [Login to your Keycloak instance](#logging-into-your-keycloak-instance) 
+1. [Login to Keycloak](#logging-into-your-keycloak-instance), which for non-external Keycloak instances is [the base URL of your running Entando application](../getting-started/README.md#configure-access-to-your-cluster) followed by `/auth/`, e.g. http://YOUR-HOST-NAME/auth. In a standard Entando installation, the base URL can be verified with `kubectl get ingress/default-sso-in-namespace-ingress`.
```
Comment (jyunmitch) [inline vuepress/docs/next/docs/consume/identity-management.md:34]: if we want to keep it simple, I think kubectl get ingress or kubectl describe ingress works too.

**Review by jyunmitch — APPROVED** [review]

File: vuepress/docs/next/docs/consume/identity-management.md (hunk 1 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/docs/consume/identity-management.md:34]: @nshaw do you have a preference for this command?

Comment (nshaw) [reply vuepress/docs/next/docs/consume/identity-management.md:34]: @Lyd1aCla1r3 The specific ingress that maps in the /auth/ path is the default-sso-in-namespace-ingress. I think the extra information is good so I'd probably keep what we have here, at the risk of it aging over time.

**Comment by nshaw** [comment]
@Lyd1aCla1r3 Looks good, just need to add in v7.1 as well.

**Review by nshaw — APPROVED** [review]

**Pull request merged**